WordPress/Classes/ViewRelated/NUX/AddUsersBlogCell.m

```diff
@@ -78,7 +78,7 @@ - (void)layoutSubviews
     y = (rowHeight - cellBlavatarSide)/2.0;
     _blavatarImage.frame = CGRectIntegral(CGRectMake(x, y, cellBlavatarSide, cellBlavatarSide));
     NSURL *blogURL = [NSURL URLWithString:_blavatarUrl];
-    [_blavatarImage setImageWithBlavatarUrl:[blogURL host]];
+    [_blavatarImage setImageWithSiteIcon:[blogURL host]];
```

Comment:
I was wondering about the same thing. Let's separate that to a new PR, once this is merged in I can take care of that :)